src/components/RecipesNav.astro

```diff
@@ -0,0 +1,30 @@
+---
+import { englishPages } from '~/content';
+import { isRecipeEntry } from '~/content/config';
+import { getLanguageFromURL } from '~/util';
+import CardsNav from './NavGrid/CardsNav.astro';
+
+export interface Props {
+	minimal?: boolean;
+}
+
+const lang = getLanguageFromURL(Astro.url.pathname);
+const recipes = englishPages.filter(isRecipeEntry);
+---
+
+<div>
+	<CardsNav
+		minimal={Astro.props.minimal}
+		links={recipes.map(({ data, slug }) => ({
+			title: data.title,
+			description: data.description,
+			href: '/' + slug.replace('en/', `${lang}/`) + '/',
+		}))}
```

Comment:
This doesn’t use the translated `title` and `description`. The approach worked for other nav components because the title was always a static service name e.g. “Netlify”, “@astrojs/react”, “Storyblok” or “Gatsby”. I’ll need to tweak this logic to pull in the translated strings while still falling back to English for untranslated pages.